src/Maker/MakeUser.php

```diff
@@ -169,13 +169,22 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         // C) Generate a custom user provider, if necessary
         if (!$userClassConfiguration->isEntity()) {
             $userClassConfiguration->setUserProviderClass($generator->getRootNamespace().'\\Security\\UserProvider');
+
+            $useStatements = [
+                UnsupportedUserException::class,
+                UsernameNotFoundException::class,
```

Comment:
I think this is backwards: `UsernameNotFoundException` is deprecated and `UserNotFoundException` is the new one.

Reply:
good catch